pipeliner/metadata/schemas.json

```diff
@@ -0,0 +1,4274 @@
+{
+  "data": {
```

Comment:
everything here is lower case, but the test for the Read method actually uses upper case: https://github.com/amp-labs/connectors/pull/661 (in `test/pipeliner/read/main.go`, line 17 is `objectName = "Contacts"`). So you either need to modify the test to use lower case "contact" and verify that it still works with the pipeliner API, or this scraper needs to modify to return upper case object names. Rest of the code here looks good, you can merge once you've made the 2 consistent.